pyproject.toml

```diff
@@ -49,6 +49,12 @@ codecov = ">=2.1.12"
 pytest-cov = "^4.1.0"
 pre-commit = ">=3.3.2"
 
+[tool.poetry.group.docs]
+optional = true
+
+[tool.poetry.group.docs.dependencies]
+mkdocs = "*"
```

Comment:
I believe this addition without a related `poetry.lock` update would cause issues with the readthedocs.org builds. Consider updating the lockfile with any new dependencies using `poetry lock --no-update`. On adding the changes to this branch via that file, the readthedocs.org builds should succeed.

Reply:
More context for observability here:
- I tested the build and found the error here: https://readthedocs.org/projects/pycytominer-test-docs/builds/21795757/.
- On updating the lockfile I saw the build succeed here: https://readthedocs.org/projects/pycytominer-test-docs/builds/21795961/

Reply:
@d33bs In the poetry redesign, I tested and later removed the `poetry lock` pre-commit hook that would have prevented this error. The main issue was that if too much time passed between making a commit and pushing it to server the pre-commit hook would pass on a local machine and then fail during the precommit github actions pipeline because of dependency changes in the intervening time. I was worried that would be too confusing to contributors. 

Perhaps the absence of the poetry lock hook is the more confusing problem though.

Reply:
Thanks! I updated the poetry lock file in 7f1aa97ea76661c6620be0284fcc0e8653c99bcb

Reply:
@gwaybio great, thanks! @kenibrewer - I've experienced similar challenges with the pre-commit hook for poetry lockfiles and feel the same. It can be confusing to figure out how to pass that specific check, making the incoming changes from contributors possibly more challenging to receive. It could possibly be nice to have `poetry lock --check` added to a GitHub Actions job in order to make this more visible. This might result in similar confusion but with the ability to enable the contributor(s) to reach a PR without pre-commit challenges, then make adjustments during review. In the meantime or in addition to, perhaps a rule of thumb here could work for maintainers, something like: "if there's a change in the `pyproject.toml` dependencies, we should expect / wonder about a change in the `poetry.lock` file as well".